docs/index.md

```diff
@@ -1,5 +1,89 @@
-are enumerated below:
+Steps to set up an analysis pipeline to analyze base editor validation experiments to run [CRISPResso2](https://github.com/pinellolab/CRISPResso2): <br/>
+    1. Using the [BEV tool](https://gpplims.broadinstitute.org/screening/bev/create) on GPP LIMS (available to GPP only) ([Jump to steps](#running-bev-tool-on-gpp-lims)) <br/>
+    2. On your local machine ([Jump to steps](#running-crispresso2-on-local-machine))
+
+## Running BEV tool on GPP LIMS
+
+**Step 1: Sample name**
+
+Assign a descriptive name for the samples, without spaces and beginning with your initials (similar to PoolQ). 
+
+**Step 2: Construct file and Barcode file**
+
+* If sequencing available through WalkUp sequencing, you can use the Walk-Up Sample Finder to access sequencing files by inputting the sample ID and lane number.
+* If sequencing files are no longer available through WalkUp, the backed up files can also be accessed by providing a file path '/rnai/screening/pooled/sequenced/\[insert_filename_here.fastq.gz\]'
+* The BEV tool demultiplexes files for you.
+
+![Folder Structure](images/folder_structure.png)
+
+**Step 3: Conditions file**
+
+A .csv file with two columns without headers: <br/>
+    1. Barcode sequences <br/>
+    2. Descriptive sample names without spaces or special characters (e.g. A549_RDA569_RepA_D7_Dropout) 
+
+**Step 4: Batch file**
+
+Batch file for a CRISPResso2 run is a .txt file with various columns indicating different parameters for each sample 
+in the run. The batch file for running CRISPResso2 on BEV is similar to the one used for running CRISPResso2 on a local machine, with some differences (bolded). Please refer to the **"[Sequence_Orientation_Documentation.html](Sequence_Orientation_Documentation.html)"** to input the appropriate amplicon and guide sequences. 
+
+The different columns in a batch file are as described below:
+1. `name`: Generate a name of the format, <font color="#8b0000"> BEV_samplenumber_primerpair</font>
+    for each sample. For example, <font color="blue">the name for sample 1 would be BEV_001_F1_A1_R1_A1, where 001 is 
+    the sample number and F1_A1 is the forward primer and R1_A1 is the reverse primer</font>.
+2. **`identifier`**: Sample name from conditions file 
+3. `amplicon_seq`: Relevant amplicon sequence for each sample. It is recommended to include the forward and the 
+   reverse primers in the amplicon sequence. For more information on how to obtain the right amplicon sequence, refer 
+   to  **"[Sequence_Orientation_Documentation.html](Sequence_Orientation_Documentation.html)"**.
+4. `guide_seq`: Sequence of sgRNA for each sample.
+5. **Other parameters**: The following parameters can be included as columns in the batch file if they are 
+    different for each sample. If they are the same for every sample, they can just be included in the syntax while 
```

Comment:
In all places replace sample with condition since we are using sample to indicate an experiment.